src/components/PoolingWidget/index.tsx

```diff
@@ -30,6 +30,8 @@ import { maxAmountsForSpread, log } from 'utils'
 import { DEFAULT_PRECISION } from 'const'
 import { Link } from 'react-router-dom'
 
+const LIQUIDITY_TOKEN_LIST = new Set(['USDT', 'TUSD', 'USDC', 'PAX', 'GUSD', 'DAI'])
```

Comment:
Add `sUSD` to the list?

Reply:
Good one! https://github.com/gnosis/dex-react/pull/499/commits/2483e63ce5f4ee3088331003b530d5de4430751e